app/main.py

```diff
@@ -1,3 +1,54 @@
-def shop_trip():
-    # write your code here
-    pass
+import app.json_parser
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    data = app.json_parser.read_json_from_file("app/config.json")
+    fuel_price = app.json_parser.get_fuel_price(data)
+
+    customers = [
+        Customer(customer_data)
+        for customer_data
+        in app.json_parser.get_customers_data(data)
+    ]
+    shops = [
```

Comment:
Rewrite using unpacking `**customer`
You can place Car creation inside `Customer.__init__`